src/main/java/net/sf/jabref/model/entry/BibEntry.java

```diff
@@ -611,4 +612,21 @@ public void addKeywords(List<String> keywords) {
         // TODO Auto-generated method stub
         return fields;
     }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) {
+            return true;
+        }
+        if (o == null || getClass() != o.getClass()) {
```

Comment:
Since I do not want to include subclasses, these should implement a proper equals method themselves.